### ISSUE TITLE
concat function is not working properly

### ISSUE THREAD

**Issue by snr-lab** [KittyMac/Sextant#7]
input
{
   "x": "1",
   "Y": 2
 }
 
 jsonpath
 $.concat($.x,"_",$.y)
 
 expected output 
 1_2
 
 actual output
 1_

**Comment by KittyMac** [comment]
should be fixed in v0.4.27

input
```
{"x":"1","Y":2,"z":true}
```
 path
```
$.concat($.x,"_",$.Y,"_",$.z)
```

output
```
1_2_true
```

**Comment by snr-lab** [comment]
Thank you @KittyMac
